src/main/java/lab5/test/players/OmolaTest.java

```diff
@@ -0,0 +1,35 @@
+package players;
+
+import game.Board;
+import game.Position;
+import game.PlayerToken;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OmolaTest {
+
+    @Test
+    void testOmolaWinsIfPossible() {
+        Board board = new Board("XXXO____O"); // Omola (X) can win by playing at (2,0)
+        Omola omola = new Omola();
+        Position move = omola.pickNextMove(board);
+        assertEquals(new Position(2, 0), move);
+    }
+
+    @Test
+    void testOmolaBlocksIfOpponentWinsNextTurn() {
+        Board board = new Board("XOXO_OX__"); // Opponent (O) can win at (1,2), so Omola should block
+        Omola omola = new Omola();
+        Position move = omola.pickNextMove(board);
+        assertEquals(new Position(1, 2), move);
+    }
+
+    @Test
+    void testOmolaPicksFirstAvailableIfNoImmediateThreat() {
+        Board board = new Board("XOXOXO___"); // No winning/blocking move, Omola should pick (2,0)
+        Omola omola = new Omola();
+        Position move = omola.pickNextMove(board);
+        assertEquals(new Position(2, 0), move);
```

Comment:
The next move is X, and there are two winning moves for X here, so again, this is just another test that a winning move is selected, not a test of what happens if there is no winning or blocking move